assignments/array-methods.js

```diff
@@ -58,28 +58,59 @@ const runners = [
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs. Combine both the first and last names and populate a new array called `fullNames`. This array will contain just strings.
 let fullNames = [];
+
+runners.forEach(function(name){
+return fullNames.push(`${name.first_name} ${name.last_name}`)
```

Comment:
Slight indentation- the return line should have one indentation inside the function '' '